templates/cloud_trial_end_warning.mjml

```diff
@@ -1,48 +0,0 @@
-<mjml>
-  <mj-head>
-    <mj-include path="./partials/style.mjml" />
-  </mj-head>
-  <mj-body css-class="emailBody" width="660px">
-    <mj-wrapper mj-class="email" border-radius="0">
-      <mj-section padding="24px" border-top="1px solid #E5E5E5" border-left="1px solid #E5E5E5" border-right="1px solid #E5E5E5">
-        <mj-column>
-          <mj-image mj-class="logo"  src="{{.Props.SiteURL}}/static/images/logo_email_dark.png" />
-        </mj-column>
-      </mj-section>
-      <mj-section padding="0 60px" border-left="1px solid #E5E5E5" border-right="1px solid #E5E5E5">
-        <mj-column>
-          <mj-text css-class="title" font-family="Open Sans, sans-serif" padding="10px 0px">
-            {{.Props.Title}}
-          </mj-text>
-          <mj-text  padding-bottom="24px"  font-weight=400 color="#3F4350" font-size="16px" line-height="24px"  color="#000000" font-family="Open Sans, sans-serif" >
-            {{.Props.SubTitle}}
-          </mj-text>
-          <mj-button href="{{.Props.ButtonURL}}" padding="0px" border-top="16px" font-weight="400 !important" css-class="button-cloud" font-family="Open Sans, sans-serif">
-            {{.Props.Button}}
-          </mj-button>
-        </mj-column>
-      </mj-section>
-      <mj-section  padding-top="57px" border-left="1px solid #E5E5E5" border-right="1px solid #E5E5E5">
-        <mj-column padding="0px">
-          <mj-image src="{{.Props.SiteURL}}/static/images/trial-ending-soon.png" width="320px" padding="0px" />
```

Comment:
Should we remove this or any other image referenced in the emails?

Reply:
Yes. I had completely forgot. Made a PR here https://github.com/mattermost/mattermost-webapp/pull/10671